models/Freelance.js

```diff
@@ -38,24 +44,31 @@ const Freelance = exports.Freelance = new mongoose.Schema({
 Freelance.pre('save', function (next) {
 
 	//review between 0 and 5
-	if (this.avgScore > 5) {
-		this.avgScore = 5;
-	} else if (this.avgScore < 0) {
-		this.avgScore = 0;
-		//maybe a problem with default -1 value of review, to check later.
+	if ( this.avgScore !== undefined ){
```

Comment:
Could you please format the code in the usual way like it was before? `if (this.avgScore !== undefined) {`